src/framed/std/core.cljc

```diff
@@ -121,10 +122,11 @@
   ([f y x]
    (f x y)))
 
-(defmacro future-loop
-  "Execute body repeatedly within a future, returning the future"
-  [& body]
-  `(future
-     (loop []
-       ~@body
-       (recur))))
+#?(:clj
+  (defmacro future-loop
```

Comment:
I often see people include clojure macros in clojurescript by having the cljs require a clj file with the macros. I'd like to learn more about including macros in cljc files. Is there documentation on this?